src/msg/async/frames_v2.h

```diff
@@ -55,7 +55,7 @@ enum class Tag : __u8 {
 struct segment_t {
   // TODO: this will be dropped with support for `allocation policies`.
   // We need them because of the rx_buffers zero-copy optimization.
-  static constexpr __le16 DEFERRED_ALLOCATION{0x0000};
+  static constexpr __le16 PAGE_SIZE_ALIGNMENT{4096};
```

Comment:
Maybe just use `CEPH_BUFFER_ALLOC_UNIT` for the magic. I'm not sure we want introducing an abstraction over page size, especially as all our targets use 4k.